### ISSUE TITLE
oidc_scopes are ignored during configuration

### ISSUE THREAD

**Issue by sergeytrasko** [hashicorp/vault-plugin-auth-jwt#44]
Seems that configuration of `oidc_scopes` attribute is ignored during configuration.

Test case:

Setup:
```
$ vault auth enable oidc
$ vault write auth/oidc/config oidc_discovery_url="https://myco.auth0.com/" oidc_client_id="id" oidc_client_secret="secret"
$ vault write auth/oidc/role/demo ttl=1h user_claim="id_sub" groups_claim="groups" allowed_redirect_uris="http://localhost:8200/ui/vault/auth/oidc/oidc/callback" oidc_scopes="test"
```

Verify:

```
$ vault read auth/oidc/role/demo
Key                      Value
---                      -----
allowed_redirect_uris    [http://localhost:8200/ui/vault/auth/oidc/oidc/callback]
bound_audiences          <nil>
bound_cidrs              <nil>
bound_claims             <nil>
bound_subject            n/a
claim_mappings           <nil>
groups_claim             groups
max_ttl                  0s
num_uses                 0
period                   0s
policies                 <nil>
role_type                oidc
ttl                      1h
user_claim               id_sub
```

**Observed:** no `oidc_scopes` attribute is found in response

**Expected:** `oidc_scopes` attribute is returned

Vault version:

```
$ vault version
Vault v1.1.0 ('36aa8c8dd1936e10ebd7a4c1d412ae0e6f7900bd')
```

**Comment by kalafut** [comment]
Hi. The value is stored and used, but wasn't being reported on read.  This was fixed in 1.1.1.

**Comment by sergeytrasko** [comment]
Indeed, tested with 1.1.1 - works fine.
Thanks!

Closing this issue.
